apps/service_define/lib/define_web/live/edit_data_definition.ex

```diff
@@ -0,0 +1,61 @@
+defmodule DefineWeb.EditDataDefinition do
```

Comment:
Is this guy in addition to the new react thing? Is he still used?

Reply:
removed